Cartridges/int_checkoutcom_sfra/cartridge/client/default/js/productDetail.js

```diff
@@ -1,8 +1,8 @@
 'use strict';
```

Comment:
i don't think we should add this file to our integration. Please can you look into removing this file also.